jdbcSource/src/test/java/io/vantiq/extsrc/jdbcSource/TestJDBC.java

```diff
@@ -906,35 +933,49 @@ public void doAsynchronousProcessing(boolean useCustomTaskConfig) throws Interru
 
         // Setup a VANTIQ JDBC Source, and start running the core
         setupSource(createSourceDef(true, useCustomTaskConfig));
-
         // Create Type to store query results
         setupAsynchType();
 
         // Create Topic used to trigger Rule
         setupTopic();
 
         // Create Procedure to publish to VANTIQ Source
-        setupProcedure();
+        setupProcedure(limitValue);
 
         // Create Rule to query the VANTIQ Source
         setupRule();
-
+    
```

Comment:
Tests periodically left detritus around, so verify that it's gone to avoid chasing red herrings